packages/mobile/src/analytics/selectors.ts

```diff
@@ -0,0 +1,79 @@
+import { createSelector } from 'reselect'
+import { nameSelector } from 'src/account/selectors'
+import { currentLanguageSelector } from 'src/app/reducers'
+import { numberVerifiedSelector } from 'src/app/selectors'
+import { backupCompletedSelector } from 'src/backup/selectors'
+import { getLocalCurrencyCode } from 'src/localCurrency/selectors'
+import { userLocationDataSelector } from 'src/networkInfo/selectors'
+import { defaultCurrencySelector } from 'src/stableToken/selectors'
+import {
+  tokensByCurrencySelector,
+  tokensByUsdBalanceSelector,
+  totalTokenBalanceSelector,
+} from 'src/tokens/selectors'
+import { Currency } from 'src/utils/currencies'
+import { accountAddressSelector, walletAddressSelector } from 'src/web3/selectors'
+
+export const getCurrentUserTraits = createSelector(
+  [
+    walletAddressSelector,
+    accountAddressSelector,
+    defaultCurrencySelector,
+    nameSelector,
+    userLocationDataSelector,
+    currentLanguageSelector,
+    totalTokenBalanceSelector,
+    tokensByUsdBalanceSelector,
+    tokensByCurrencySelector,
+    getLocalCurrencyCode,
+    numberVerifiedSelector,
+    backupCompletedSelector,
+  ],
+  (
+    walletAddress,
+    accountAddress,
+    currency,
+    name,
+    { countryCodeAlpha2 },
+    language,
+    totalBalanceUsd,
+    tokensByUsdBalance,
+    tokensByCurrency,
+    localCurrencyCode,
+    hasVerifiedNumber,
+    hasCompletedBackup
+  ) => {
+    const currencyAddresses = new Set(
+      Object.values(tokensByCurrency).map((token) => token?.address)
+    )
+
+    // Don't rename these unless you have a really good reason!
+    // They are used in users analytics profiles + super properties
+    return {
+      accountAddress,
+      walletAddress,
+      currency, // TODO: is this the right name in multi token context?
```

Comment:
What is property meant to be used for? The `defaultCurrencySelector` was just used to decide the initial selected token in the legacy send flow but will go away soon probably. Can we just remove this?

Reply:
I copied what we were already sending.
It was useful to target the "default" currency used by others, but now in a multi token context I'm not sure anymore.